scripts/setup-web2py-debian.sh

```diff
@@ -25,17 +25,20 @@ read CONFIRM
 echo "installing useful packages"
 echo "=========================="
 apt-get update
+apt-get -y install sudo
 apt-get -y install ssh
 apt-get -y install zip unzip
 apt-get -y install tar
 apt-get -y install openssh-server
 apt-get -y install build-essential
+apt-get -y install python
+#apt-get -y install python2.5
 apt-get -y install ipython
```

Comment:
```suggestion
apt-get -y install ipython3
```

Reply:
never tried web3py with ipython3, had someone tried it ?